ui/src/organizations/containers/OrganizationView.tsx

```diff
@@ -196,6 +204,7 @@ class OrganizationView extends PureComponent<Props> {
                 <GetOrgResources<Variable[]>
                   organization={org}
                   fetcher={getVariables}
+                  orderBy={{keys: ['name']}}
```

Comment:
should orderBy be an optional prop with default= `{keys:['name]}`

Reply:
@ebb-tide I was wondering that after most places it was used benefited from a simple name ordering. It should just ignore ordering if the collection doesn't have that key. Hmm would it be worth typing the orderedBy.keys property names of the collection item?